### ISSUE TITLE
Update Design challenge html template

### ISSUE THREAD

**Issue by sushilshinde** [topcoder-platform/tc-elasticsearch-feeder-service#99]
Formatting should look that used to look in v3

[V3 template.zip](https://github.com/topcoder-platform/tc-elasticsearch-feeder-service/files/2740883/V3.template.zip)

Currently in v4 it looks like this 

https://www.topcoder.com/challenges/30078581

**Comment by sushilshinde** [comment]
Original issue reported here https://github.com/topcoder-platform/community-app/issues/1860

**Comment by sushilshinde** [comment]
fyi @ajefts @cwdcwd

**Comment by skyhit** [comment]
@ajefts @cwdcwd @sushilshinde originally, it is formalized like https://github.com/topcoder-platform/tc-elasticsearch-feeder-service/blob/dev/src/main/java/com/appirio/service/challengefeeder/manager/ChallengeDetailFeederManager.java#L136-L148

which format should we follow?  I can make the change. or we should return field separately, and let community app to control the format, but this is conflicting our purpose to have unified detail description field.

**Comment by sushilshinde** [comment]
No, we need to add HTML formatting in the templating. I will give you the template.

**Comment by skyhit** [comment]
@sushilshinde share the original template? we don't have actually, previously, it is part of the UI work, let's define some?

**Comment by sushilshinde** [comment]
detailedRequirements value is being built by code
https://api.topcoder.com/v4/challenges/30078581

that's what I'm referring to @skyhit

**Comment by sushilshinde** [comment]
By adding a placeholder for the fixed text in their returned HTML

This is dynamic code generated in api
<this is the place holder for fix text>
This is dynamic code generated in api

or generate complete HTML on in API only

@skyhit

**Comment by sushilshinde** [comment]
@skyhit this is old template 

![screenshot 2019-01-24 17-57-51](https://user-images.githubusercontent.com/683231/51678323-03f63000-2002-11e9-906a-aa4779265735.png)

**Comment by sushilshinde** [comment]
@skyhit there are still few issues with the fix
1. Challenge Introduction/Challenge Summary is not being returned as part of introduction field 
http://api.topcoder-dev.com/v4/challenges/30055072
```
introduction: "<h3 class='roundLabel'>Round 1</h3> <h3 class='roundLabel'>Round 2</h3>",
```
Comapared to v3 

http://api.topcoder-dev.com/v4/challenges/30055072
```
introduction: "This is challenge introduction",
round1Introduction: "Round 1 info",
round2Introduction: "round 2 info",

```
I think it should be returned as 

```
introduction: "<p>This is challenge introduction</p><h3 class='roundLabel'>Round 1</h3> <h3 class='roundLabel'>Round 2</h3>",
```

1. It shouldn’t show the Round headings if it’s not a multi-round challenge:  https://test-community-app.topcoder-dev.com/challenges/30054531


https://test-community-app.topcoder-dev.com/challenges/30054531
http://api.topcoder-dev.com/v4/challenges/30055052
